runtime/vm/rasdump.c

```diff
@@ -649,7 +649,9 @@ J9RASCheckDump(J9JavaVM* javaVM)
 #if defined(LINUX) || defined(AIXPPC)
 	IDATA rc;
 	U_64 limit;
```

Comment:
While you're here, please add explicit initial values for these two locals.

Reply:
updated